src/encoded/audit/experiment.py

```diff
@@ -262,11 +262,16 @@ def audit_experiment_missing_unfiltered_bams(value, system, files_structure):
 def audit_experiment_with_uploading_files(value, system, files_structure):
     if files_structure.get('original_files'):
         for file_object in files_structure.get('original_files').values():
-            if file_object['status'] in ['uploading', 'upload failed', 'content error']:
+            if file_object['status'] in ['upload failed', 'content error']:
                 detail = 'Experiment {} '.format(value['@id']) + \
                     'contains a file {} '.format(file_object['@id']) + \
                     'with the status {}.'.format(file_object['status'])
                 yield AuditFailure('file validation error', detail, level='INTERNAL_ACTION')
+            if file_object['status'] in ['uploading']:
```

Comment:
This should be an else if since the conditions are mutually exclusive.  
```
elif file_object['status'] in ['uploading'] 
```
and also get rid of the list in the conditional 
```
elif file_object['status'] == 'uploading'
```
 If we need to add values later on then we'll make it a list then.

Reply:
Fixed.